genienlp/metrics.py

```diff
@@ -344,184 +173,6 @@ def computeNMTBLEU(outputs, targets):
     return bleu_metric.compute(predictions=outputs, references=targets)['bleu'] * 100
 
 
-class Rouge(Rouge155):
-    """Rouge calculator class with custom command-line options."""
-
-    # See full list of options here:
-    # https://github.com/andersjo/pyrouge/blob/master/tools/ROUGE-1.5.5/README.txt#L82
-    DEFAULT_OPTIONS = [
-        '-a',  # evaluate all systems
-        '-n',
-        4,  # max-ngram
-        '-x',  # do not calculate ROUGE-L
-        '-2',
-        4,  # max-gap-length
-        '-u',  # include unigram in skip-bigram
-        '-c',
-        95,  # confidence interval
-        '-r',
-        1000,  # number-of-samples (for resampling)
-        '-f',
-        'A',  # scoring formula
-        '-p',
-        0.5,  # 0 <= alpha <=1
-        '-t',
-        0,  # count by token instead of sentence
-        '-d',  # print per evaluation scores
-    ]
-
-    def __init__(self, n_words=None, keep_files=False, options=None):
-
-        if options is None:
-            self.options = self.DEFAULT_OPTIONS.copy()
-        else:
-            self.options = options
-
-        if n_words:
-            options.extend(["-l", n_words])
-
-        stem = "-m" in self.options
-
-        super(Rouge, self).__init__(n_words=n_words, stem=stem, keep_files=keep_files)
-
-    def _run_rouge(self):
-        # Get full options
-        options = ['-e', self._rouge_data] + list(map(str, self.options)) + [os.path.join(self._config_dir, "settings.xml")]
-
-        # logging.info("Running ROUGE with options {}".format(" ".join(options)))
-        # print([self._rouge_bin] + list(options))
-        pipes = Popen([self._rouge_bin] + options, stdout=PIPE, stderr=PIPE)
-        std_out, std_err = pipes.communicate()
-
-        div_by_zero_error = std_err.decode("utf-8").startswith("Illegal division by zero")
-        if pipes.returncode == 0 or div_by_zero_error:
-            # Still returns the correct output even with div by zero
-            return std_out
-        else:
-            raise ValueError(std_out.decode("utf-8") + "\n" + std_err.decode("utf-8"))
-
-
-def computeROUGE(greedy, answer):
```

Comment:
Please bring back Rouge, perhaps using HF dataset. It is not just for summarization, it is also used in many natural_seq2seq tasks like paraphrasing. I know it is not on by default, but we do use it from time to time.